hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -327,3 +327,48 @@ def test_manual_remove_element(self):
         assert (
             sorted(s._xray_markers.keys()) ==
             ['Zn_Ka', 'Zn_La'])
+
+
+@lazifyTestClass
```

Comment:
OK, then rebinning of lazy signals will need some special care and will be for another day/PR!!
I think this is all good now?! (I feel like, it is not the first time I said this in this PR...)